jaeger_client/config.py

```diff
@@ -138,42 +148,42 @@ def _validate_config(self, config):
                              format(','.join(unexpected_config_keys)))
 
     @property
-    def service_name(self):
+    def service_name(self) -> str:
         return self._service_name
 
     @property
-    def metrics(self):
-        return self._metrics
+    def metrics(self) -> Optional[Metrics]:
+        return self._metrics  # type:ignore
```

Comment:
I don't think there is such thing as `self._metrics`, let's return None and add a comment "Deprecated"

Reply:
done